src/main/java/org/jenkinsci/plugins/github/webhook/subscriber/DuplicateEventsSubscriber.java

```diff
@@ -0,0 +1,137 @@
+package org.jenkinsci.plugins.github.webhook.subscriber;
+
+import static com.google.common.collect.Sets.immutableEnumSet;
```

Comment:
No need for Guava here? https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/EnumSet.html#of(E,E...)

Reply:
(or plain `Set.of`)

Reply:
wrote after following the other two subscriber implementations,
* https://github.com/jenkinsci/github-plugin/blob/d2e9e60422ae6f72b6ead117b7ea7a5f8a34470d/src/main/java/org/jenkinsci/plugins/github/webhook/subscriber/DefaultPushGHEventSubscriber.java#L54-L60
* https://github.com/jenkinsci/github-plugin/blob/d2e9e60422ae6f72b6ead117b7ea7a5f8a34470d/src/main/java/org/jenkinsci/plugins/github/webhook/subscriber/PingGHEventSubscriber.java#L48-L54